packages/ketcher-react/src/script/ui/dialog/toolbox/labelEdit.module.less

```diff
@@ -0,0 +1,64 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+@import '../../../../style/mixins.less';
+@import '../../../../style/variables.less';
+
+.labelEdit {
+  display: flex;
+  white-space: nowrap;
+
+  form {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    padding-top: 12px;
+
+    label {
+      text-align: center;
+    }
+
+    input {
+      margin-left: 8px;
+      min-width: 140px;
+    }
+  }
+
+  footer {
+    padding: 16px 12px 12px;
+
+    .okButton {
```

Comment:
with buttonsNameMap prop we won't need it
also, you can use corresponding mixins when needed

Reply:
deleted